.github/workflows/analyze-modified-files.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         continue-on-error: false
         if: env.diff != '' && matrix.task == 'flake8'
         run: |
-          flake8 --count --select=E9,F63,F7,F82 --show-source --statistics ${{ env.diff }}
+          flake8 --count --select=E9,F63,F7,F82 --ignore F824 --show-source --statistics ${{ env.diff }}
```

Comment:
I wonder why `--select` is using `=` and `--ignore` isn't. It looks like it works either way.

Reply:
It feels more natural to me without `=`, so that's what i typed, and i didnt think about changing any of the existing arguments.